### DISCUSSION TITLE
1.20 Wishlist

### DISCUSSION THREAD

**Discussion by emielderckx** [PurpurMC/Purpur#1256] [Feature Requests]
Just comment your 1.20 requests below, and we'll add them to the list. This mega thread is for **1.20 only**, not for 1.19.4.

- Make Gold trim block piglin anger.
- Make Turtle helmet with Netherite/Gold/Iron/Diamond trim also have the ability's of those armor pieces.
- Pandas eat bamboo wood houses.

**Comment by granny** [comment]
added an `mobs.piglin.ignores-armor-with-gold-trim` option starting with build 2034 https://github.com/PurpurMC/Purpur/commit/cf3dca9776afbdf8a9568689eca4e66281d0e0bf
